internal/characters/itto/skill.go

```diff
@@ -60,34 +58,24 @@ func (c *char) Skill(p map[string]int) action.ActionInfo {
 		if done {
 			return
 		}
-		con := &ushi{
-			src:    c.Core.F,
-			expiry: c.Core.F + 360, // 6 seconds from hit/land
-			char:   c,
-		}
-		c.Core.Constructs.New(con, true)
 		done = true
+		// spawn ushi. on-field for 6s
+		c.Core.Constructs.New(c.newUshi(6*60), true)
+
+		// Energy. 33% chance of 3 particles
```

Comment:
The actual chance is 50% for 3 particles and 50% for 4 particles.
https://library.keqingmains.com/evidence/characters/geo/itto?q=itto